src/engine/include/models/mesh_frame.hpp

```diff
@@ -79,13 +80,18 @@ class MeshFrame
      * Returns bounding box (AABB).
      * Total length: 8
      */
-    Vector3 *getBoundingBox() { return boundingBox; };
+    Vector3 *getBoundingBox() { return boundingBoxObj->getVertices(); };
```

Comment:
We have a slight discrepancy here in the name of the function and what it does. I think that it should just return reference to `boundingBox` and `getBoundingBoxP()` will removed?